products/cdk/download.adoc

```diff
@@ -47,8 +50,12 @@ To get started using the Red Hat CDK:
 * Follow the instructions in the link:https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/version-7/container-development-kit-installation-guide[Container Development Installation Guide] to install the CDK.
 * Read link:https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/version-7/getting-started-with-container-development-kit/[Getting Started with Container Development Kit] to learn what is included with the CDK and how to use it.
 
-[.callout-light]
-*NOTE:*  In order to run the Red Hat Enterprise Linux Vagrant box, you must have signed the Developer Suite terms and conditions. If you have not done this already, log into the Customer Portal with your Red Hat account and link:https://www.redhat.com/wapps/ugc[sign the Red Hat Enterprise Linux Developer Suite Terms & Conditions].
+Follow our Get Started Guides to build "Hello, World" in a container using your choice of language:
+
+* #{site.download_manager_base_url}/products/cdk/get-started-cdk2-nodejs/[Node.js]
+* #{site.download_manager_base_url}/products/cdk/get-started-cdk2-php/[PHP]
+* #{site.download_manager_base_url}/products/cdk/get-started-cdk2-python/[Python]
+* #{site.download_manager_base_url}/products/cdk/get-started-cdk2-ruby/[Ruby]
```

Comment:
@adelasofia - Good catch, Thanks!  

(I didn't notice because that content is already on stage and hasn't changed.)